clang-tools-extra/clang-tidy/bugprone/UseAfterMoveCheck.cpp

```diff
@@ -35,6 +37,11 @@ struct UseAfterMove {
 
   // Is the order in which the move and the use are evaluated undefined?
   bool EvaluationOrderUndefined;
```

Comment:
While you're at it, please initialize `EvaluationOrderUndefined` to `false` as well.

Reply:
sure. but for the reason explained at https://github.com/llvm/llvm-project/pull/93623#discussion_r1631614642. but please note, it's just for the sake of readability / consistency, not for the correctness.